modules/users/client/views/profile/profile-view.client.view.html

```diff
@@ -172,7 +172,7 @@
                   ui-sref="profile.experiences.new({username: profileCtrl.profile.username})"
                 >
                   <i class="icon-plus-squared-alt"></i>
-                  Write a reference
+                  Share your experience
```

Comment:
this, again, does not change the title on mobile, but I guess we can merge this and update it later?

Reply:
Yep, correct, let's handle the mobile navigation in one swoop separately.